Samples/GPSTracker/GPSTracker.Common/AzureHelper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using Microsoft.WindowsAzure.ServiceRuntime;
+
+namespace GPSTracker.Common
+{
+    /// <summary>
+    /// Helper class to isolate from Microsoft.WindowsAzure.ServiceRuntime.dll and its failures to load outside of Azure
+    /// </summary>
+    public class AzureHelper
```

Comment:
nit: 'Helper' is a poor naming convention that can become an unstructured catchall for junk code.  Can you be more explicit about what this is class is?  Maybe AzureDeployment?  DeploymentEnvironment? ..